From: Mourad De Clerck <bugs-debian@aquazul.com>
Date: Wed, 25 Nov 2020 13:55:28 +0100
Subject: profile.d: Disable gvfs plugins when listing flatpak installations

This avoids gvfs-daemon being started when logging in as root via ssh.

Bug-Debian: https://bugs.debian.org/975710
Forwarded: https://github.com/flatpak/flatpak/pull/4049
Applied-upstream: 1.9.4
---
 profile/flatpak.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/profile/flatpak.sh b/profile/flatpak.sh
index 6c6e113..9dc6cf9 100644
--- a/profile/flatpak.sh
+++ b/profile/flatpak.sh
@@ -5,7 +5,7 @@ if command -v flatpak > /dev/null; then
         (
             unset G_MESSAGES_DEBUG
             echo "${XDG_DATA_HOME:-"$HOME/.local/share"}/flatpak"
-            flatpak --installations
+            GIO_USE_VFS=local flatpak --installations
         ) | (
             new_dirs=
             while read -r install_path
